near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -346,8 +374,10 @@ impl NonFungibleToken {
         let approved_account_ids =
             if self.approvals_by_id.is_some() { Some(HashMap::new()) } else { None };
 
+        if let Some((id, storage_usage)) = initial_storage_usage {
```

Comment:
hi, why not refund to the signer if no refund id is provided?

Reply:
The idea is that if you pass a `None`, no refund should happen.  The point is that you want to have the ability to mint multiple tokens in one transaction and then do a refund after adding them all to storage.

Reply:
> The point is that you want to have the ability to mint multiple tokens in one transaction and then do a refund after adding them all to storage.           

yeah this makes sense to me.

But I would prefer to refund to the txn signer *by default* in case no refund id is given, rather than keeping all the extra deposit in the contract

Reply:
That's what happens in the guarded mint. How else would you specify no refund? Add another argument?